docs/standard/serialization/system-text-json/customize-properties.md

```diff
@@ -245,35 +245,33 @@ By default, properties are serialized in the order in which they're defined in t
 
 ## Use GitHub Copilot to customize property names and order
 
-You can use GitHub Copilot in your IDE to generate code to customize names and order of serialized properties. You can customize the prompt to output a JSON string with property names and values that suit your requirements.
+You can use GitHub Copilot with `System.JSON.Text` attributes to apply patterns of changes to how your code serializes without having to explicitly specify the change in each one.
 
-The following example shows you how to use Copilot to modify existing code to customize property names and order when serializing to JSON.
+Suppose you want to change how properties are serialized—formatting property names with underscores between words and ordering them in reverse alphabetical order, while keeping the original property name. You can use Copilot Chat to make these changes with a prompt like this (this prompt works best with the Claude 3.5 Sonnet model).
 
-1. Add the following C# example code to a code file `Example.cs` in your editor.
-   In Visual Studio, you can use a C# console application project to try this example.
-
-   :::code language="csharp" source="snippets/how-to-6-0/csharp/copilot-example.cs":::
-
-   `Example.cs` code does the following:
-
-   - Creates an instance of the `Person` class and initializes its properties with values.
-   - Serializes the `person` object to a JSON string using `JsonSerializer.Serialize`.
-   - Prints the following JSON string to the console:
-
-   ```json
-   {"FirstName":"John","LastName":"Doe","Age":30,"Country":"USA"}
-   ```
-
-1. In Copilot Chat, enter the following prompt to modify the code to customize names and order of the JSON serialization output.
+```copilot-prompt
+Update class <your class name>: when the property name contains more than one word, change the serialization name for that property to have underscores between each word.
+Serialize the properties in reverse alphabetical order without changing the order in which the properties are listed.
+```
 
-    ```copilot-prompt
-    #Example.cs modify code to use System.Text.Json to customize property names and order of JSON output from serialization.
-    Set property names: FirstName to first_name, LastName to last_name.
-    Set order to: Country, FirstName, LastName, Age.
-    Provide customized serialization output.
-    ```
+Here's a more complete version of the example that includes a simple class.   
+ 
+```copilot-prompt 
+Take this C# class: 
+public class WeatherForecast    
+{                  
+public DateTime Date { get; set; }         
+public int TemperatureC { get; set; }                 
+public int TemperatureF { get; set; }                
+public string? Summary { get; set; }                  
+public int WindSpeed { get; set; }     
```

Comment:
Why is the code not formatted correctly? I always format things correctly, even in prompts.

```suggestion
    public DateTime Date { get; set; }
    public int TemperatureC { get; set; }
    public int TemperatureF { get; set; }
    public string? Summary { get; set; }
    public int WindSpeed { get; set; }
```

Reply:
Thanks. Rushed PR for the discussion.